### PR TITLE
Remove CSINodeInfo feature gate

### PR DESCRIPTION
**Pull request by ialidzhikov**
**What type of PR is this?**
/kind cleanup
/sig storage

**Does this PR introduce a user-facing change?**:

```release-note
The `CSINodeInfo` feature gate that is GA since v1.17 is unconditionally enabled, and can no longer be specified via the `--feature-gates` argument.
```

### CODE REVIEW

File: pkg/scheduler/framework/plugins/nodevolumelimits/non_csi.go [hunk 1]
```diff
@@ -512,10 +511,7 @@ func getMaxEBSVolume(nodeInstanceType string) int {
 	return volumeutil.DefaultMaxEBSVolumes
 }
 
-// getCSINodeListerIfEnabled returns the CSINode lister or nil if the feature is disabled
+// getCSINodeListerIfEnabled returns the CSINode lister
```
Comment (Huang-Wei) [inline pkg/scheduler/framework/plugins/nodevolumelimits/non_csi.go:514]:
```suggestion
// getCSINodeLister returns the CSINode lister
```

**Review by Huang-Wei — COMMENTED** [review]
A nit. LGTM otherwise on scheduling side.

File: pkg/scheduler/framework/plugins/nodevolumelimits/non_csi.go (hunk 1 above)
Comment (ialidzhikov) [reply pkg/scheduler/framework/plugins/nodevolumelimits/non_csi.go]: Thank you, addressed in https://github.com/kubernetes/kubernetes/compare/78b49ee8024d99ae5a849cabac7b05a68a4d4e87..e6baa0d43c4c8ad5ca894f1a7327776915d87b22.

**Comment by Huang-Wei** [comment]
/approve
on scheduling side.

**Pull request merged**